build.gradle

```diff
@@ -32,7 +32,7 @@ subprojects {
         mysqlDriverVersion = '8.4.0'
         postgresqlDriverVersion = '42.7.5'
         oracleDriverVersion = '23.6.0.24.10'
-        sqlserverDriverVersion = '12.8.1.jre8'
+        sqlserverDriverVersion = '12.9.0.jre8-preview'
```

Comment:
Not sure whether to use the preview version...

Reply:
We discussed that before and [concluded we should not](https://github.com/scalar-labs/scalardb/pull/2481#issuecomment-2611718186). 
The only way to ignore a specific version for a Dependabot group update is to add this version to ignore `12.9.0.jre8-preview` to the `dependabot.yaml`, which is a bit troublesome. 
I was hoping the stable version `12.9.0.jre8` would be released quickly but since it is not the case, I will resort to update the `dependabot.yaml`

Reply:
I see. Thanks!

Reply:
https://github.com/scalar-labs/scalardb/pull/2517